.github/actions/javascript/contributorChecklist/contributorChecklist.js

```diff
@@ -127,8 +127,17 @@ function getAllComments() {
     }, response => _.map(response.data, comment => comment.body));
 }
 
+function countCheckedBoxes(string) {
+    return string.toLowerCase().match(/\[x\]/g).length;
```

Comment:
I would also check for `X` uppercase x since I saw people use that and then not knowing why the check fails.

Reply:
since this uses `toLowerCase()` it should check for both

Reply:
completely missed that. Thanks